plugin/pkg/admission/limitranger/admission.go

```diff
@@ -249,6 +249,42 @@ func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.Resourc
 		}
 	}
 
+	for i := range pod.Spec.InitContainers {
+		container := &pod.Spec.InitContainers[i]
```

Comment:
https://github.com/kubernetes/kubernetes/pull/23567/files/805cb1e4b870ef2f22562afd5ecd559c82909659..c3dad6ac86987ff44c2eb12a4707e7636803e48d#diff-e3f052839796781484de133fea5d5b5dL217  -- isnt that the same code block in that loop?